src/modelbench/run.py

```diff
@@ -269,23 +269,25 @@ def calibrate(update: bool, file) -> None:
 
 def update_standards_to(standards_file):
     reference_suts = [
-        ModelGaugeSut.for_key("mistral-7b"),
+        ModelGaugeSut.for_key("gemma-2-9b-it-hf"),
+        ModelGaugeSut.for_key("llama-3.1-8b-instruct-turbo-together"),
     ]
     if not ensure_ensemble_annotators_loaded():
         print("Can't load private annotators needed for calibration")
         exit(1)
 
-    benchmarks = [GeneralPurposeAiChatBenchmark()]
-    for l in Locale:
+    benchmarks = []
+    for l in [Locale.EN_US]:
         benchmarks.append(GeneralPurposeAiChatBenchmarkV1(l, "ensemble"))
-    run_result = run_benchmarks_for_suts(benchmarks, reference_suts, 100)
+    run_result = run_benchmarks_for_suts(benchmarks, reference_suts, 2000)
```

Comment:
I'm curious about the reasoning behind 2000 max items

Reply:
Ah, sorry, I shouldn't have left that in. The calibration set should be all the items, but I was doing practice runs with smaller numbers. 2000 is just larger than the 1000 max. I'll change it so it's always everything.